templates/document.html

```diff
@@ -183,6 +183,15 @@ <h3 data-template="lang:translate">series</h3>
                             <p data-template="app:series">WeGA, Briefe, Digitale Edition</p>
                         </div>-->
                         
+                        <div data-template="app-shared:if-exists" data-template-key="authors">
+                            <h3 data-template="lang:translate" class="media-heading">authors</h3>
+                            <ul class="inline-list">
+                                <li data-template="app-shared:each" data-template-from="authors" data-template-to="author">
+                                    <span data-template="app-shared:print" data-template-key="author"/>
```

Comment:
`app-shared:print()` will only print the atomic value. Use `app-shared:output()` instead to directly output the HTML  ;)

Reply:
:man_facepalming: